en,zh: release note for 1.4.2 (#2201)
pingcap/docs-tidb-operator: en/releases/release-1.4.2.md

---
title: TiDB Operator 1.4.2 Release Notes
---

# TiDB Operator 1.4.2 Release Notes

> Release date: February 2, 2023

Author: ```suggestion
Release date: February 3, 2023
```

TiDB Operator version: 1.4.2

## Bug fix

> - Fix the issue that TiFlash not listen on IPv6 addr when the `preferIPv6` config is enabled ([#4850](https://github.com/pingcap/tidb-operator/pull/4850), [@KanShiori](https://github.com/KanShiori))

Reviewer: ```suggestion
- Fix the issue that TiFlash not listen on IPv6 address when the `preferIPv6` config is enabled ([#4850](https://github.com/pingcap/tidb-operator/pull/4850), [@KanShiori](https://github.com/KanShiori))
```

> - Fix the issue that TiFlash not listen on IPv6 address when the `preferIPv6` config is enabled ([#4850](https://github.com/pingcap/tidb-operator/pull/4850), [@KanShiori](https://github.com/KanShiori))

Reviewer 2: ```suggestion
- Fix the issue that TiFlash does not listen on IPv6 addresses when the `preferIPv6` configuration is enabled ([#4850](https://github.com/pingcap/tidb-operator/pull/4850), [@KanShiori](https://github.com/KanShiori))
```
